packages/maskbook/src/plugins/Wallet/services/keystore.ts

```diff
@@ -0,0 +1,143 @@
+import scrypt from 'scrypt-js'
```

Comment:
Please use https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API.

Reply:
this is not resolved.

Reply:
web crypto don't support scrypt